src/classes/i18n4Js.php

```diff
@@ -44,6 +45,7 @@ private function getTerms(): array
             'generic-delete-warning' => _('Delete this?'),
             'link-delete-warning' => _('Delete this link?'),
             'new-apikey-warning' => _("This is the only time the key will be shown! Make sure to copy it somewhere safe as you won't be able to see it again:"),
+            'no-self-links' => _('No self links!'),
```

Comment:
"The link was not created because it points to itself." or "Sorry, you cannot create a link that points to the present entry", or something similar. "No self links!" seems a bit raw/dry.

BTW, interesting read: https://ux.stackexchange.com/questions/31359/should-error-messages-apologize